ludwig/hyperopt/execution.py

```diff
@@ -31,6 +31,7 @@
 logger = logging.getLogger(__name__)
 
 try:
+    import kubernetes
```

Comment:
This is not a listed dependency. We should also not make it required, even for ray users, as not everyone will be running on k8s.

Reply:
Fixed.